cumulus/client/collator/src/lib.rs

```diff
@@ -69,14 +69,14 @@ where
 	RA::Api: CollectCollationInfo<Block>,
 {
 	/// Create a new instance.
-	fn new(
+	pub fn new(
```

Comment:
Actually revert all the changes in this file.

Reply:
Done.